app/src/main/java/com/github/se/signify/ui/screens/home/ASLRecognition.kt

```diff
@@ -45,7 +45,7 @@ import com.google.mediapipe.tasks.components.containers.NormalizedLandmark
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun ASLRecognition(
-    handLandMarkViewModel: HandLandMarkViewModel,
+    handLandMarkViewModel: HandLandmarkViewModel,
```

Comment:
```suggestion
    handLandmarkViewModel: HandLandmarkViewModel,
```
You can proceed with the rest of the changes

Reply:
Done.